dot-net-sdk/EppoClient.cs

```diff
@@ -394,52 +394,50 @@ public static EppoClient Init(EppoClientConfig eppoClientConfig)
     {
         InputValidator.ValidateNotBlank(flagKey, "Invalid argument: flagKey cannot be blank");
 
-        // If no actions are given - a valid use case - return the `defaultValue`.
-        bool isBanditFlag = _configurationStore.GetBanditFlags().IsBanditFlag(flagKey);
-
-        if (actions.Count == 0 && isBanditFlag)
-        {
-            // If not graceful mode, raise an exception here?
-            return new(defaultValue);
-        }
-
-
         // Get the user's flag assignment for the given key.
         var variation = GetStringAssignment(
             flagKey,
             subject.Key,
             subject,
             defaultValue);
 
-        // If not a bandit, return the computed String assignment
-        if (!isBanditFlag)
-        {
-            Logger.Warn($"[Eppo SDK] Flag \"{flagKey}\" does not contain a Bandit");
-            return new(variation);
-        }
-
-        try
+        // Only proceed to computing a bandit if there are actions provided.
+        if (actions.Count > 0)
```

Comment:
It's actually cleaner to process this way instead of checking `isBanditFlag`

Reply:
Nit: I think it's easier to handle the special case here and then proceed with the standard case, e.g.
```suggestion
        if !(actions.Count > 0) {
           return ...
        }
        continue...
```

Reply:
👍